### ISSUE TITLE
dataio - urwid detection error is ambiguous (Trac #930)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#924]
clarify the warning:

```text
-- +-- dataio-pyshovel *not* included (urwid python package < 1.1.0)
```

Migrated from https://code.icecube.wisc.edu/ticket/930
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T21:44:52", 
    "description": "clarify the warning:\n\n{{{\n-- +-- dataio-pyshovel *not* included (urwid python package < 1.1.0)\n}}}", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547243092626431", 
    "component": "cmake", 
    "summary": "dataio - urwid detection error is ambiguous", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-04-14T19:50:30", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150417T00:28:36: 
* nega commented:

> In changeset 2132:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2132"
> reword "urwid too old" msg.
> 
> closes #924
> ```


* nega changed status from "new" to "closed"
* nega changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:44:52: nwhitehorn commented:

> In changeset 169137:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169137"
> reword "urwid too old" msg.
> 
> closes #924
> ```
